app-frontend/config/webpack/environments/development.js

```diff
@@ -21,11 +21,17 @@ module.exports = function (_path) {
             'history-api-fallback': true,
             port: port,
             proxy: {
-                '*': 'http://localhost:9100'

```

Comment:
huh, so I guess the only change that was really required was making the frontend always request from "/" instead of a full url.